### ISSUE TITLE
Flume Integration no longer working when we moved from a PI to a NUC

### ISSUE THREAD

**Issue by gdeuss** [home-assistant/home-assistant.github.io#21442]
### Feedback

When trying to add the Flume integration to the home assistant in now returns "Unknown Error occurred" when trying to link the flume account to home assistant  

<img width="838" alt="image" src="https://user-images.githubusercontent.com/95149025/152255827-cc5e570c-41f7-4900-a943-7872ff607272.png">

### URL

https://www.home-assistant.io/integrations/flume/

### Version

2022.2.0

### Additional information

<img width="838" alt="image" src="https://user-images.githubusercontent.com/95149025/152255869-77e6ac1d-def1-456d-9182-6485a3cc6cca.png">

**Comment by bdraco** [comment]
This repo is for documentation issues.  Please open an issue here

https://github.com/home-assistant/core/issues/new/choose

Thanks 👍

**Comment by bdraco** [comment]
Also please provide logs in the new issue as we won't be able to help without them.
Thanks

**Comment by gdeuss** [comment]
Nick,

The issue was resolved as follows:

When adding the Flume integration I noticed that I had the option to add Flume or another instance of Flume. I did not notice that before. When I selected Flume I either got the error msg I told you after a while or later I got an “Abort” instantly. No idea why the error changed later.
When I selected the “Add another instance” the Flume integration came back instantly and the dashboard I had previously set up, worked again immediately. The only off part is that the Flume integration is not currently showing on my integrations tab, see picture below. The data is updating fine however….

Not sure why that is happening, attached the log anyways could not find anything about Flume in it, the other errors have been addressed

***@***.***





Gustav Deuss
GRD Energy, Inc.
1 Avenue of the Palms, Suite 10C
San Francisco, CA 94130
O 415 891 8824
M 650 291 2569
